webapp/lib/app/nook/view.dart

```diff
@@ -44,8 +48,11 @@ class NookPageView extends PageView {
     conversationListPanelView = new ConversationListPanelView();
     conversationPanelView = new ConversationPanelView();
     replyPanelView = new ReplyPanelView();
+    notesPanelView = new NotesPanelView();
     tagPanelView = new TagPanelView();
```

Comment:
nit: same as above, let's keep tags next to replies and notes as a slightly more separate thing now
```suggestion
    tagPanelView = new TagPanelView();
    notesPanelView = new NotesPanelView();
```